containers/ddev-webserver/files/etc/skel/.bashrc

```diff
@@ -111,3 +111,11 @@ if ! shopt -oq posix; then
     . /etc/bash_completion
   fi
 fi
+
+# set COMPOSER_BIN_DIR to $WEBSERVER_DOCROOT/vendor/bin if unset
+if [[ ! -v $COMPOSER_BIN_DIR ]]; then
```

Comment:
I haven't used `-v` before. I would have expected `if [ -d ${COMPOSER_BIN_DIR} ]` here?

Reply:
The intention was: If the variable is unset (so that users could still use a different directory for the composer bin dir) set it to a sensible default (/var/www/html/vendor/bin).

Reply:
I think `-z` is the normal usage?

Reply:
There are quite a few ways to do this tbh, -v Is just what I usually do as it just checks if a variable is set.

```
-v varname

    True if the shell variable varname is set (has been assigned a value).
```

( From https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html )